CHANGELOG.unreleased.md

```diff
@@ -11,7 +11,9 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.released
 [Commits](https://github.com/scalableminds/webknossos/compare/25.01.0...HEAD)
 
 ### Added
+- Added the possibility to configure a rotation for a dataset, which can be toggled off and on when viewing and annotating data. [#8159](https://github.com/scalableminds/webknossos/pull/8159)
 - Added the possibility to use the "merger mode" even when the user has annotated volume data in the current layer (as long as no other mapping is active). [#8335](https://github.com/scalableminds/webknossos/pull/8335)
```

Comment:
This needs to be removed. Likely a small merging mistake


```suggestion
- Added the possibility to use the "merger mode" even when the user has annotated volume data in the current layer (as long as no other mapping is active). [#8335](https://github.com/scalableminds/webknossos/pull/8335)
```

Reply:
whoops, I failed to catch that, thanks!

Reply:
(I think it was maybe due to the reversion of the dataset rotation PR? because in the current master, both entries are in the unreleased changelog? nvm, I think updating this branch will clear this up anyways :) )